climpred/smoothing.py

```diff
@@ -250,13 +250,11 @@ def _reset_temporal_axis(ds_smoothed, smooth_kws={'time': 4}, dim=None):
         raise ValueError('smooth_kws doesnt contain a time dimension.', smooth_kws)
     smooth = list(smooth_kws.values())[0]
     dim = list(smooth_kws.keys())[0]
-    if dim == 'lead':
-        composite_values = ds_smoothed[dim].values
-    elif dim == 'time':
-        # TODO: This assumes that smoothing is over years. Is this fair?
+    try:
+        # TODO: This assumes that smoothing is only done in years. Is this fair?
```

Comment:
for goddard2013 yes, but in general this shouldnt be assumed. in the future a keyword should be passed or the keyword should be taken from lead.attrs.units

Reply:
Okay, another issue to open. We can't solve all of time in this PR. Thanks